packages/mongodb-artifact-generator/src/commands/generateDocsMetaDescription.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  createConfiguredAction,
+  withConfig,
+  withConfigOptions,
+} from "../withConfig";
+import { createCommand } from "../createCommand";
+import { makeRunLogger, type RunLogger } from "../runlogger";
+import PromisePool from "@supercharge/promise-pool";
+import { makeGenerateMetaDescription } from "../docs-meta-descriptions/generateMetaDescription";
+import { urlToFilename } from "../utils";
+
+let logger: RunLogger;
+
+type GenerateDocsMetaDescriptionCommandArgs = {
+  runId?: string;
+  llmMaxConcurrency: number;
+  url: URL[];
+};
+
+export default createCommand<GenerateDocsMetaDescriptionCommandArgs>({
+  command: "generateDocsMetaDescription",
+  builder(args) {
+    return withConfigOptions(args)
+      .option("runId", {
+        type: "string",
+        demandOption: false,
+        description:
+          "A unique name for the run. This controls where outputs artifacts and logs are stored.",
+      })
+      .option("url", {
+        type: "string",
+        demandOption: true,
+        description:
+          "The URL of the page to generate a meta description for. It must be a valid URL that exists in the EAI knowledge base.",
+        coerce: (url: string | string[]) => {
+          const urls = Array.isArray(url) ? url : [url];
+          return urls.map((url) => {
+            try {
+              return new URL(url);
+            } catch (e) {
+              throw new Error(`Invalid page URL: ${url}`);
+            }
+          });
+        },
+      })
+      .option("llmMaxConcurrency", {
+        type: "number",
+        demandOption: false,
+        default: 10,
```

Comment:
i rec lower default. many LLMs could get rate limited for this on a non-trivial amount of pages. 3-5 prob better default